src/quests.json

```diff
@@ -40,7 +40,7 @@
                     "text": "By the 1840s the chaste, neoclassical architecture of the buildings, the landscaped garden, and the promenades up to and around the reservoirs on \"Fair Mount,\" as the rise was identified as early as 1687 on a map of Philadelphia, were major attractions. Ornamental sculptures, fountains, and airy gazebos added to the visual pleasure of the site.", 
                     "attribution": "Jane Mork Gibson"
                 }, 
-                "radius": 5, 
```

Comment:
Not precisely relevant to this PR, but I think we should .gitignore this file since it's generated from the grunt command.